src/polycubed/src/rest_server.cpp

```diff
@@ -931,5 +937,157 @@ const std::string &RestServer::getPort() {
     return port;
 }
 
+void RestServer::create_metrics() {
+  logger->info("loading metrics from yang files");
+  try {
+    // uri and metricHandler are not necessary, I use pistache to registry metrics
+    registry = std::make_shared<prometheus::Registry>();
+    std::vector<std::string> name_services = core.get_servicectrls_names_vector();
+
+    // loop on all services (not cubes)
+    for (size_t i = 0; i < name_services.size(); i++) {
+      
+      // for every service I get the array of InfoMetric
+      auto metrics_service = core.get_service_controller(name_services[i]).get_infoMetrics();
+      for(auto metric: metrics_service)  {
+        if (metric.typeMetric == "C") {
```

Comment:
What does it mean "C"? Shouldn't we replace with an enum/define?

Reply:
I do not know if this is related to https://github.com/polycube-network/polycube/pull/313/#discussion_r462307552 but if only intends to define a #define or enum in the rest_server, yes, done :+1: